### DISCUSSION TITLE
[STUDIO]Liaison avec un acteur d'une autre scène

### DISCUSSION THREAD

**Discussion by WIToliv** [witsa/synapps#567] [Q&A]
Hello, 
Comment on fait pour liaison un bouton de navigation avec un acteur écran qui n'est pas dans la même scène ?
Merci

**Comment by Raphwit** [comment]
Hello, 
A ma connaissance, lier des acteurs inter scène n'est pas possible, il est préférable de garder les actions d'une scène à l'intérieur de celle-ci (créer un bouton de navigation à l'intérieur de la scène ou se trouve l'acteur écran).

Dans quel cas d'utilisation voudrait-on avoir un bouton de navigation en dehors de la scène dans laquelle on navigue ?

**Comment by WIToliv** [comment]
Ben en fait j'ai fait une scène(1) qui s'affiche en iframe dans ma scène principale.
c'est scène (1) comporte des boutons qui ouvriront par exemple le pdf de schéma de raccordement ou autre dans le screen de la page principale.
Mais si on peut pas faire de lien entre les scènes ben je vais ramener mes boutons sur la scène principale.

**Comment by Raphwit** [comment]
Effectivement, il est préférable de garder les actions qui agissent sur les éléments d'une scène dans cette même scène, dans le cas contraire, cela pourrait créer des confusions pour l'utilisateur ( par exemple si l'élément qui est contrôlé n'est pas affiché ).

Je laisse l'issue ouverte en attendant une réponse définitive d'Emmanuel d'ici la semaine prochaine !

**Comment by WIToliv** [comment]
Alors je reviens sur mon issue.
J'ai regardé comme était fait le modèle en 3 parties sur synapps.
Et le mien est un peu pareil, la partie de droite viens d'une autre scène "informations". Il y a les boutons d'"actions".
Ceux ci ouvrent une nouvelle fenêtre (chez moi c'est l'ouverture des schémas). Moi ce que j'aurai voulu c'est que ces boutons ouvrent ces fichiers dans l'écran principale car la synapp va tourner sur un écran tactile et que l'on ne peut pas faire ouvrir de multi fenêtre. Donc pour ce faire, je dois avoir les boutons sur la scène principale et ne pas faire comme ce modèle ?

**Comment by Raphwit** [comment]
Le mécanisme d'ouverture ne sera pas exactement le même en fait.

Dans la synapp d'exemple, on ouvre simplement le fichier hors de la Synapp.

En revanche, si je comprend bien votre besoin, il faut dans votre cas ouvrir le fichier directement dans la synapp ?
Cela engendre une petite question, connaissez vous le type de fichier que vous aller ouvrir (pdf/une image png, etc ..) ?

**Comment by WIToliv** [comment]
oui dans la synapps principale comme une nouvelle scène
en fait j'ai créé une scène ou on ouvre le pdf comme pour l'affichage de la chaufferie etc...
dans mon cas c'est des fichiers pdf

**Comment by edejean** [comment]
Bonjour
Désolé de commenter cette vieille discussion, mais ouvrir un PDF dans une synapps est maintenant possible ? J'en étais rester au bug qui nous demandait de télécharger le pdf plutôt que l'afficher ?!?
Cdt
Eric
